### ISSUE TITLE
compute-baseline: incomplete import statements?

### ISSUE THREAD

**Issue by tidoust** [web-platform-dx/web-features#967]
The NPM `compute-baseline` package does not work on my machine. Node.js complains about import statements with `ERR_MODULE_NOT_FOUND` errors. As far as I can tell, that's because the import statements in the package do not specify the filename extensions whereas [that's mandatory](https://nodejs.org/docs/latest/api/esm.html#mandatory-file-extensions). Also, there's an attempt to import a directory, which is not supported either.

**Comment by ddbeck** [comment]
This is a consequence of using `tsx` to run everything here (and elsewhere I've used this code)—I never see what it looks like as a plain Node.js import. Oops!

I think the steps to fix this are:

- [ ] Add `.js` extensions to all the imports
- [ ] Change `moduleResolution` to `"NodeNext"` (maybe, need to test this)

**Comment by foolip** [comment]
@tidoust can you check if v0.8.1 works for you?

**Comment by tidoust** [comment]
v0.8.1 is `web-features`, not `compute-baseline`, right?

Testing compute-baseline by creating a dependency on a local folder with the latest version of the code in the web-features repository, I still have an import problem: 

> TypeError [ERR_IMPORT_ASSERTION_TYPE_MISSING]: Module "[full path redacted]/@mdn/browser-compat-data/data.json" needs an import attribute of type "json"

That's triggered by `import bcd from "@mdn/browser-compat-data"` statements such as:
https://github.com/web-platform-dx/web-features/blob/main/packages/compute-baseline/src/browser-compat-data/browser.ts#L1C1-L5C35

These import statements need something like `with { type: 'json' }` (well, or `assert { type: 'json'}` depending on the runtime version).

**Comment by ddbeck** [comment]
https://github.com/web-platform-dx/web-features/pull/1051

**Comment by foolip** [comment]
> v0.8.1 is `web-features`, not `compute-baseline`, right?

Right... I got myself confused because I saw the hopeful fix for this in the list of changes for web-features.

**Comment by tidoust** [comment]
Closing as I believe this issue was addressed a long time ago through #1027 and #1051. I confirm package now works with Node.js without problem.
